t/data/engine/parse_json/01/translate.serge

```diff
@@ -0,0 +1,20 @@
+jobs
+{
+    {
+        @inherit                                 ../../common.serge#job_template
+
+        source_match                             \.json$
+        key_separator                            .
+        key_leading_separator                    NO
```

Comment:
Whether or not a key should start with a leading separator: `/part/item` = YES `part.item` = NO.

The fact that you need to ask suggests the name should be changed.